broadinstitute/pearl
# AR-109 - configurable footer capability

Footers can now be added to a portal by specifying an HtmlSection.  OurHealth footer added based on https://dspuxteam.invisionapp.com/console/Our-Health-cl6fbib5i1cpq01zec5qx922m/cle98lbk70ji201bo7m8u6emh/play. Styling is rough-and-ready (and not themed), just wanted to get something in that we can refine as we learn more about, e.g. where the privacy policy and terms of use go.

TO TEST:
1. restart adminApiApp and ApiParticipantApp
2. run `./scripts/populate_portal.sh ourhealth`
3. confirm footer appears as below
![image](https://user-images.githubusercontent.com/2800795/223134972-376c2562-93f6-4f07-9577-a32caa0781b7.png)
